src/java.desktop/share/classes/sun/swing/SwingAccessor.java

```diff
@@ -47,6 +56,26 @@ public final class SwingAccessor {
     private SwingAccessor() {
     }
 
+    public static AccessibleComponentAccessor getAccessibleComponentAccessor() {
+        return accessibleComponentAccessor;
+    }
+
+    public static void setAccessibleComponentAccessor(final AccessibleComponentAccessor accessibleAccessor) {
+        accessibleComponentAccessor = accessibleAccessor;
+    }
+
+    /**
+     * This interface provides access to the renderer's accessibility component.
+     * For example, the renderer of a list element, a table cell, or a tree node
+     */
+    public interface AccessibleComponentAccessor {
```

Comment:
Note that this class declares interfaces at the top, and then field/get/set at the bottom of the file.

Reply:
done